test/integration_tests/test_end_to_end.py

```diff
@@ -0,0 +1,127 @@
+import json
+from pathlib import Path
+from time import sleep
+
+import fnmatch
+
+import paramiko
+import pytest
+
+from app.rabbit_context import RabbitContext
+from config import TestConfig
+
+ICL1E_message_template = {
```

Comment:
🎨two spaces after import statements 

```suggestion

ICL1E_message_template = {
```